code/tINIT/removeDrugReactions.m

```diff
@@ -0,0 +1,22 @@
+function outModel=removeDrugReactions(inModel)
+% outModel=removeDrugReactions(inModel)
+%
+% Removes reactions from Human-GEM related to drugs
```

Comment:
I wonder if it would be easier to just remove all reactions that are in the `Drug metabolism` subsystem. I imagine it might remove a few additional reactions, but unlikely that they would be important for anything. Something to think about in the future.

Reply:
Aha, I never thought about that.

Reply:
As a review, I have following concerns to this function:
- there is some commented lousy code
- it doesn't appear to be generic enough for future use, thus might be okay to be hard coded inside `prepHumanModelForftINIT`

Reply:
Yeah, I'm not sure that should be there - this is to show how I tested that the code works. Maybe we could clean that up.